TLS.md

```diff
@@ -58,7 +58,7 @@ To connect to this Valkey server with `valkey-cli`:
         --key ./tests/tls/valkey.key \
         --cacert ./tests/tls/ca.crt
 
-This will disable TCP and enable TLS on port 6379. It's also possible to have
+Specifying `port 0` will disable TCP and enable TLS on port 6379. It's also possible to have
```

Comment:
Just specifying `port 0` doesn't enable TLS on port 6379.

"This" referred to the whole example above, with `--tls-cert-file` and all that.

```suggestion
Specifying `port 0` will disable TCP. It's also possible to have
```

Reply:
Thank you for the clarification, @zuiderkwast, I was searching for an explanation for `port 0`. My change was based on the configuration described in the  [TLS Redis Docs](https://redis.io/docs/latest/operate/oss_and_stack/management/security/encryption/) Attached is a screenshot :
 
<img width="751" alt="Captura de pantalla 2024-05-02 a las 9 37 05 a  m" src="https://github.com/valkey-io/valkey/assets/38367844/2a64b6ac-8e42-4f60-861f-e5e63420e588"> 
Let me know if there's a way to incorporate this in TLS.md.

Reply:
Thanks for the clarification, your suggestion looks good.